backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/controllers/SiteMapController.java

```diff
@@ -0,0 +1,86 @@
+package org.molgenis.emx2.web.controllers;
+
+import static org.molgenis.emx2.SelectColumn.s;
+import static org.molgenis.emx2.web.MolgenisWebservice.getSchema;
+
+import com.redfin.sitemapgenerator.ChangeFreq;
+import com.redfin.sitemapgenerator.WebSitemapGenerator;
+import com.redfin.sitemapgenerator.WebSitemapUrl;
+
+import java.net.MalformedURLException;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+import org.molgenis.emx2.MolgenisException;
+import org.molgenis.emx2.Row;
+import org.molgenis.emx2.Schema;
+import org.molgenis.emx2.Table;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+
+public class SiteMapController {
```

Comment:
I would prefer if this follows same style and naming as https://github.com/molgenis/molgenis-emx2/blob/master/backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/JsonYamlApi.java with a create option

Reply:
combining a 'create' function with a static class does not make any sense to be , JsonYamlApi.create function seams to add certain routes to the spark route request handler, as the order in which the routes added matter hiding these adds in a create function makes it hard to follow the request flow.